src/lib/engine/dgraph.js

```diff
@@ -23,3 +23,14 @@ export function dependencyMap(dGraph: DGraph): Array<Object>{
   return _.flatten(asLists)
 }
 
+// The bizarro graph is the version of the graph where a guesstimates input is
+// replaces its guesstimte
+export function toBizarroGraph(graph, guesstimateForm){
```

Comment:
Comment seems like it is messed up somehow and this name is very noninformative.